src/components/movieList/MovieList.js

```diff
@@ -1,69 +1,55 @@
-import React, { Fragment, useEffect, useState } from "react";
+import React, { Fragment, useEffect} from "react";
```

Comment:
Directory name "movieList" doesn't match the naming convention.

Reply:
A component in React should always use a pascal case. I changed movieList to MovieList in my last commit.